src/aosm/azext_aosm/cli_handlers/onboarding_core_vnf_handler.py

```diff
@@ -72,7 +72,7 @@ def _get_processor_list(self) -> List[AzureCoreArmBuildProcessor | VHDProcessor]
             arm_input = ArmTemplateInput(
                 artifact_name=arm_template.artifact_name,
                 artifact_version=arm_template.version,
-                default_config=None,
+                default_config={"imageName": self.config.nf_name + "Image"},
```

Comment:
What happens if default_config is passed in and gives `imageName` a different value?

Reply:
it can't, we were previously passing no default config. But in this specific case of azure core vnfs, we are now giving imageName. This doesnt come from anywhere else except us giving it here